### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by ggabernet**
Many thanks for contributing to nf-core/bcellmagic!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If necessary, also make a PR on the [nf-core/bcellmagic branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/bcellmagic)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated
- [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/bcellmagic/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by ggabernet** [comment]
Hi @apeltzer and @FriederikeHanssen ,

this PR should contain basically the commits that you've already reviewed to the `dev` branch from `dsl2` 😄

**Review by apeltzer — APPROVED** [review]
Guess this is the same + the requested changes so good to go from my side 👍🏻

**Review by FriederikeHanssen — APPROVED** [review]
yes same as Alex :)  👍

**Review by apeltzer — APPROVED** [review]

**Comment by apeltzer** [comment]
Maybe release date July 19th now, otherwise looking good 👍🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,95 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2021-07-16 "Lumos"
```
Comment (ggabernet) [inline CHANGELOG.md:6]:
```suggestion
## [2.0.0] - 2021-07-19 "Lumos"
```

**Pull request merged**